packages/components/src/InputTime/InputTime.rebuilt.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { ChangeEvent, useId } from "react";
+import { useTimePredict } from "./hooks/useTimePredict";
+import { InputTimeProps, InputTimeRebuiltProps } from "./InputTime.Types";
+import { dateToTimeString, timeStringToDate } from "./utils/input-time-utils";
+import {
+  FormFieldProps,
+  FormFieldWrapper,
+  useFormFieldWrapperStyles,
+} from "../FormField";
+
+export function InputTimeRebuilt({
+  defaultValue,
+  value,
+  onChange,
+  ...params
+}: InputTimeRebuiltProps) {
+  const { setTypedTime } = useTimePredict({
+    value,
+    handleChange,
+  });
+  console.log("version2");
+
+  const { inputStyle } = useFormFieldWrapperStyles(params);
+
+  const id = getId(params);
+  const fieldProps = {
+    onChange: handleChange,
+    ...(defaultValue && {
+      defaultValue: dateToTimeString(defaultValue),
+    }),
+    ...(!defaultValue && { value: dateToTimeString(value) }),
+  };
```

Comment:
this object doesn't make much sense anymore. we need to manage the default value/value logic in a different way since previously this was something we'd make use of inside FormField's RHF logic